### PR TITLE
Status draft does not display on my artifact

### PR DESCRIPTION
**Pull request by ckouder**
should fix issue 153. Now MyArtifact page should look like:

image

The first one is a draft, second one is an published artifact

### CODE REVIEW

File: components/ArtifactShort.vue [hunk 1]
```diff
@@ -267,6 +261,15 @@ export default {
...
         : false
     },
+    isDraft() {
+      let hasPublicationAttr = (this.artifact.artifact_group.publications !== undefined)
+      hasPublicationAttr && this.artifact.artifact_group.publications.forEach(publication => {
+        if (publication.artifact_id == this.artifact.id) {
+          return false;
+        }
+      });
+      return hasPublicationAttr && this.artifact.artifact_group.publications.length ? true : false;
```
Comment (carboxylman) [inline components/ArtifactShort.vue]: Shouldn't this line just be a `return true;` ?  Seems like a given artifact is only a draft if is not already in the publications list (e.g. if it is not caught by the check on line 267).

File: pages/artifact/_artifact_group_id.vue [hunk 2]
```diff
@@ -69,6 +69,19 @@ export default {
...
+      // the user will have outdated artifact data when he returns from
+      // editing page. In those cases, the mounted hook is not triggered
+      // so we have to manually fetch new information
+      if (!val.query.edit && !val.query.editing) {
+        this.$store.dispatch('artifacts/fetchArtifact', {
+          artifact_group_id: this.$route.params.artifact_group_id,
+          id: this.$route.params.id
+        })
+      }
+    }
```
Comment (carboxylman) [inline pages/artifact/_artifact_group_id.vue:84]: Seems like a good catch, thanks!!

File: components/ArtifactShort.vue (hunk 1 above)
Comment (ckouder) [reply components/ArtifactShort.vue]: Yes you are right!

File: pages/artifact/_artifact_group_id.vue (hunk 2 above)
Comment (ckouder) [reply pages/artifact/_artifact_group_id.vue:84]: thank you!

**Pull request merged**